fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -113,10 +113,13 @@ protected void setUpJMSInfo(final UriInfo uriInfo, final HttpHeaders headers) {
                 baseURL = uriInfo.getBaseUri().toString();
             }
             LOGGER.debug("setting baseURL = " + baseURL);
-            session.getFedoraSession().addSessionData(BASE_URL, baseURL);
-            if (!StringUtils.isBlank(headers.getHeaderString("user-agent"))) {
-                session.getFedoraSession().addSessionData(USER_AGENT, headers.getHeaderString("user-agent"));
-            }
```

Comment:
determine if this data can be stored in the HttpSession?

Reply:
I think it makes more sense in the HttpSession.

Reply:
Decision from meeting: Skip for now. We can determine the next steps when we identify the need.